hub/src/XChainHubSingle.sol

```diff
@@ -23,6 +24,17 @@ import {IHubPayload} from "@interfaces/IHubPayload.sol";
 contract XChainHubSingle is XChainHub {
     event SetStrategyForChain(address strategy, uint16 chain);
 
+    /// @dev dont use: these are random addresses
+    address public constant ADDY_AVAX_STRATEGY =
+        0xa23bAFEB30Cc41c4dE68b91aA7Db0eF565276dE3;
```

Comment:
made up addresses? better to use addy 0x01 to make it clear it's not real

Reply:
Foundry will not run test suite with invalid addresses.